inc/config.class.php

```diff
@@ -282,7 +282,10 @@ static function loadInSession() {
           && $config->fields['use_filter_assign_group']) {
          $user = new PluginEscaladeUser();
          if ($user->getFromDBByCrit(['users_id' => $_SESSION['glpiID']])) {
-            $config->fields['use_filter_assign_group'] = $user->fields['use_filter_assign_group'];
+            //if a bypass is defined for user
+            if ($user->fields['use_filter_assign_group']) {
```

Comment:
```suggestion
            if (!$user->fields['use_filter_assign_group']) {
```
Would it be possible to not change the label if we did this ?

Reply:
By default all user have this option to fasle, if we do this, when enabled filter on general configuration, all user can bypass